pkg/planner/cardinality/BUILD.bazel

```diff
@@ -59,7 +59,7 @@ go_test(
     data = glob(["testdata/**"]),
```

Comment:
This code is too far from the master branch, it needs to be synced with the master.